src/index.ts

```diff
@@ -5,7 +5,7 @@ export { default as Breadcrumbs } from "./components/Breadcrumbs/Breadcrumbs";
 export { default as Button } from "./components/Button/Button";
 export { ButtonTypes } from "./components/Button/ButtonTypes";
 export { default as Card } from "./components/Card/Card";
-export { default as Checkbox } from "./components/Checkbox/Checkbox";
+export { default as CheckboxOld } from "./components/Checkbox-old/CheckboxOld";
```

Comment:
By doing this you're not exporting the "new" `Checkbox` component, just the `CheckboxOld` component which, if no one should use, should not be exported.